### ISSUE TITLE
Max terrain texture quality and shinny textures

### ISSUE THREAD

**Issue by wizardofid29** [Dark-Basic-Software-Limited/GameGuruRepo#1787]
Max terrain textures in some instances are too shinny, the overall quality of textures is poor.

Recommendation was to implement a method that allows tiling textures better, much like water can currently be tiled.However lee turned down this suggestion.So some thing will still need to be done with regards to the poor quality of some textures in max in particular the default grass terrain texture which isn't good enough, tiling the textures would have resolved the issue..  

![shinny textures](https://user-images.githubusercontent.com/82841733/149808080-16d53783-8177-4a94-92ab-6f3bd39b674d.jpg)

**Comment by UltraVox001** [comment]
For the usual functions of textures (Tiling, Stretching,...) you are completely right. I'm looking forward to this.
For brilliance of terrain textures, go to :
- Edit
- Settings
- Developer
- Check : Display terrain Debug Mode
- Click on the Terrain icon (top left)
- Click on Paint Terrain (top right)
- Scroll all the way down, and set the "Non Metal Reflectance" slider to 0.001
- Get out of Developer Mode and keep up your good work :)

**Comment by wizardofid29** [comment]
While this might be a fix, it doesn't fix the underlining problem terrain textures being overly shinny in the first place.Unless it is mud, wet or near a river, terrain texture by default should have almost zero shine pending the texture type, in this case a grass texture live above should have zero shininess, and nothing else.

In other words the gloss/metalness texture for this texture isn't being rendered correctly and changing the reflectance for one changes it for every texture, which isn't a desirable solution.

**Comment by UltraVox001** [comment]
Yes, it is a temporary solution for a better rendering. Normally, each texture selected in the editor should open a submenu giving access to Tiling, Stretching and Shine (at a minimum). It is not implemented at the moment. But rest assured, it will happen. TGC is aware that textures are never perfect. They always need adjustments depending on the map, the weather, the environment, etc. Be patient, it's only an Alpha and soon you will have access to all these functions.

**Comment by OldFlak** [comment]
Yes agree with wizard - we not looking for workarounds here (they be for Discord\Forms) git is for bug bashing :)

The textures as mentioned are too shiny and unless wet like mud as also mentioned by wizard should not have any shine at all.

This is either a bug where shine is being added to textures where it shouldn't be, or the textures themselves have not been created correctly. More likely the later, but perhaps a mix of both?

But whatever the reason for it is, the shine on terrain ruins MAX scenes - so it really needs sorting.
